crdppf/__init__.py

```diff
@@ -76,8 +81,8 @@ def includeme(config):
     config.add_route('getTownList', 'getTownList')
     config.add_route('getTopicsList', 'getTopicsList')
     config.add_route('createNewDocEntry', 'createNewDocEntry')
-    config.add_route('getDocumentReferences', 'getDocumentReferences')
-    config.add_route('getLegalDocuments', 'getLegalDocuments')
+    config.add_route('document_ref', 'getDocumentReferences')
+    config.add_route('legal_documents', 'getLegalDocuments')
```

Comment:
Yes... per flake8, functions do not contain any capital letters and `config.add_route` in fact references a function...

Basically, you should never use capital letters in your route names...